### ISSUE TITLE
a11y radio: tabbing interactions broken with Apple voice over

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-formkit#430]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [x] Chrome
- [x] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [x] Desktop
- [ ] Mobile

### Steps to reproduce

1. Go to the Auro doc site
2. Navigate to auro-radio
3. Turn on Apple's Voice Over
4. Navigate to radio button example for Default Group
5. Notice how it's impossible to select other options

Adversly, turn off Apple's Voice Over and attempt to complete the same task and you will be successful. 

### Expected behavior

WIth both standard keyboard access and Voice Over screen reader experiences, users should be able to complete the relative task. 

### Which version of auro-formkit do you have installed?

@current

### Reproducible on the Auro doc site

- [x] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by blackfalcon** [comment]
Unable to repro on other computers. Closing as this appears to be a localized issue.
